mybank/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>돈쪽이</title>
+</head>
+<body>
+    <header>
+        <h1>돈쪽이의 가계부</h1>
+    </header>
+    <main>
+        <section id="asset">
+            <h3>나의 자산</h3>
+            <h1>52500</h1>
+            <div id="total_inout">
+                <span class="input" id="plus_ic">+</span><span class="input">122600</span>
+                &nbsp;
+                <span class="output" id="minus_ic">-</span><span class="output">70100</span>
+            </div>
+        </section>
+        <hr>
+        <section id="nav">
+            <nav id="nav_date">
+                <button>&lt;</button>
+                <h3>10월 11일</h3>
+                <button>&gt;</button>
+            </nav>
+            <div id="nav_list">
+                <h4>내역 리스트</h4>
+                <div id="ckbox_wrapper">
+                    <input type="checkbox" id="input_box" name="input_box" checked />
+                    <label for="input_box">수입</label>
+                    <input type="checkbox" id="output_box" name="output_box" checked />
+                    <label for="output_box">지출</label>
+                </div>
+            </div>
+        </section>
+        <hr>
+        <section id="list">
+            <article class="list_wrapper">
+                <h5>식비</h5>
+                <h4>GS25 코엑스점</h4>
+                <h5 class="output">-3100</h5>
+                <button class="list_btn">x</button>
+            </article>
+            <article class="list_wrapper">
+                <h5>취미</h5>
+                <h4>양재 화훼시장</h4>
+                <h5 class="output">-12000</h5>
+                <button class="list_btn">x</button>
+            </article>
```

Comment:
```suggestion
        <section id="list">
            <ul>
            <li class="list_wrapper">
                <h5>식비</h5>
                <h4>GS25 코엑스점</h4>
                <h5 class="output">-3100</h5>
                <button class="list_btn">x</button>
            </li>
            <li class="list_wrapper">
                <h5>취미</h5>
                <h4>양재 화훼시장</h4>
                <h5 class="output">-12000</h5>
                <button class="list_btn">x</button>
            </li>
```

이렇게 ul-li로 쓰는게 적합할 것 같은데 어떻게 생각하시나용?

Reply:
맞는 것 같다구 생각합니당!